src/components/permissions/routes.ts

```diff
@@ -171,7 +171,7 @@ const routeUserPermissions = async (request: any, reply: any) => {
 }
 
 const routeUpdateRowPolicies = async (request: any, reply: any) => {
-  // TODO, add parameters to only drop specific policies, for now drop and reinstante them all
+  // TODO, add parameters to only drop specific policies, for now drop and reinstate them all
```

Comment:
Idk if this TODO should be kept. I think it's fine to reinstate all for now - specially if it's that quick!

Reply:
Just saying to remove the TODO part - but keep the comment about what this is doing

Reply:
I dunno, I'm not really changing anything here, just fixing a typo.

We need to do some more general improvements to the whole update policies area, but I'd rather not put any thought into that for this PR, that's a whole other can of worms.

Reply:
Fine, leave the TODO which isn't something we plan to do anyway

Reply:
It's not a matter of whether we do or don't plan to do anything -- it's just that it has nothing to do with this PR. (You wouldn't have even seen this pre-existing comment if I hadn't fixed a typo in it ;)